manifests/init.pp

```diff
@@ -197,10 +197,8 @@
     fail("start_delay requires at least Monit 5.0. Detected version is <${monit_version_real}>.")
   }
 
-  anchor { "${module_name}::begin": }
-  -> class { "${module_name}::install": }
+  class { "${module_name}::install": }
```

Comment:
am I reading the diff wrong or doesn't this change the logic? (but it seems to work)

Reply:
it removes the anchor resources; that in theory shouldnt change the logic as we keep the ordering of the resources

Reply:
converted it to contains and relationships as ekohl suggested